music/src/music_vae/model.ts

```diff
@@ -18,7 +18,8 @@
 import * as tf from '@tensorflow/tfjs-core';
```

Comment:
Same reviewer comments as for MusicRNN apply here as well.

Reply:
BTW I didn't add auxiliary inputs for MusicVAE because none of our models actually use them.

Reply:
To clarify, I meant that to say that lots of the requests I made above are relevant here as well, but I didn't want to point them all out :)

Reply:
Yes :)  I think because no auxiliary inputs it's just changing the chord progression default, which I fixed.